packages/gatsby/src/utils/webpack.config.js

```diff
@@ -451,47 +451,61 @@ module.exports = async (program, directory, suppliedStage) => {
   }
 
   if (stage === `build-html` || stage === `develop-html`) {
+    // Packages we want to externalize to save space
```

Comment:
Primary goal of using externals is/was to shave some time (not space) when building bundle used to produce static html files (just to provide some context why we have in the first place)